### PR TITLE
Add a description of send_dos_opportunities_email.py script to README

### PR DESCRIPTION
**Pull request by CrystalPea**
Script with description still missing:

notify-suppliers-whether-application-made-for-framework.py

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ The `scripts` folder in this repository contains scripts that interact with the
 
   Checks all free-text fields in G-Cloud services for "bad words" and generates a CSV report of any bad words found.
 
+* `send_dos_opportunities_email.py`
```
Comment (samuelhwilliams) [inline README.md]: This script uses underline notation; all other scripts in this repo appear to use dashes. Could we update the script (assuming this isn't just a typo) to conform to that standard, please?

File: README.md [hunk 2]
```diff
@@ -101,6 +101,10 @@ The `scripts` folder in this repository contains scripts that interact with the
...
+
+  **Runs Mon-Fri on Jenkins**. Send emails with new opportunities on a lot to suppliers who are on this lot.
```
Comment (samuelhwilliams) [inline README.md]: Might be nice to include the time it runs (I think it's something like 8am or 9am?).

File: README.md (hunk 1 above)
Comment (CrystalPea) [inline README.md]: @samuelhwilliams you are totally right that this name is not consonant with the rest, but if we change it, we also need to change the Jenkins job and this probably is outside of scope of this manual update. Should we add it to the tech debt?

File: README.md (hunk 2 above)
Comment (CrystalPea) [inline README.md]: Good call, I will add the runtime ☺️

File: README.md (hunk 1 above)
Comment (samuelhwilliams) [inline README.md]: If you don't think it is a quick fix, then yes we should make a record of it somewhere. Tech debt is probably the most appropriate place to put it. 👍

Comment (CrystalPea) [inline README.md]: Ok, added to tech debt: https://trello.com/c/3Ca1I5m6/152-change-filename-of-senddosopportunitiesemailpy-in-scripts

**Comment by MuriloDalRi** [comment]
I think we can merge this, no?

**Comment by CrystalPea** [comment]
I think you are right @MuriloDalRi, should I do it now?

**Comment by MuriloDalRi** [comment]
You were not here on Monday when we had a tech retro. We agreed to either merge or close PRs older than two weeks since 4th wall was getting really cluttered.

**Comment by CrystalPea** [comment]
ok, it is merged now :)

**Pull request merged**